client/src/components/README.md

```diff
@@ -1,3 +1,13 @@
-# React components
+## `<LeftAndMain />`
```

Comment:
This README file will be showed in the pattern library in the "Notes" tab. It can be use to give some guidance about how to use the component.